src/components/WorkspaceDetails/WorkspaceDetails.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import {
+  OverviewPage,
+  DetailsPage,
+  DetailsItem,
+  k8sGetResource,
+  K8sModelCommon /* , useK8sWatchResource */,
+} from '@openshift/dynamic-plugin-sdk-utils';
+import type { K8sResourceCommon } from '@openshift/dynamic-plugin-sdk-utils';
+import { useParams } from 'react-router-dom';
+
+const WorkspaceDetailsTabs = (workspace: K8sResourceCommon, loaded: boolean, error: { message: string; status: number }) => {
+  return [
+    {
+      key: 'overview',
+      title: 'Overview',
+      content: (
+        <OverviewPage loaded={loaded} resource={workspace} loadError={error} rightColumn={<></>}>
```

Comment:
Should use `OverviewTab`?  It looks like `OverviewTab` is only being used in tests.

Reply:
I'm trying to set up a pattern for future use.  In theory, there will be more than the overview tab. So if each tab has a separate file we prevent having a lot of tab-specific code on the WorkspaceDetails page.

Reply:
After we talked - I see that you are pointing out what I intended to do, but didn't actually connect the files.  The correct imports have been included.